client/authority-discovery/src/service.rs

```diff
@@ -78,7 +78,10 @@ impl Service {
 	///
 	/// Returns `None` if no entry was present or connection to the
 	/// [`crate::Worker`] failed.
-	pub async fn get_authority_id_by_peer_id(&mut self, peer_id: PeerId) -> Option<AuthorityId> {
+	pub async fn get_authority_id_by_peer_id(
```

Comment:
also needs docs update
```suggestion
	pub async fn get_authority_ids_by_peer_id(
```